src/main/java/jenkins/plugins/git/GitToolChooser.java

```diff
@@ -48,12 +52,14 @@
      * @throws IOException on error
      * @throws InterruptedException on error
      */
-    public GitToolChooser(String remoteName, Item projectContext, String credentialsId, String gitExe, Boolean useJGit) throws IOException, InterruptedException {
+    public GitToolChooser(String remoteName, Item projectContext, String credentialsId,
```

Comment:
Please add javadoc comments for the new parameters.  That will resolve the javadoc warnings below and keep the javadoc clean

Reply:
Done